pkg/standalone-utils/contracts/PauseHelper.sol

```diff
@@ -0,0 +1,153 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+pragma solidity ^0.8.24;
+
+import "@openzeppelin/contracts/utils/structs/EnumerableSet.sol";
+
+import { IVault } from "@balancer-labs/v3-interfaces/contracts/vault/IVault.sol";
+import { IVaultAdmin } from "@balancer-labs/v3-interfaces/contracts/vault/IVaultAdmin.sol";
+import { SingletonAuthentication } from "@balancer-labs/v3-vault/contracts/SingletonAuthentication.sol";
+
+contract PauseHelper is SingletonAuthentication {
```

Comment:
Should we have an IPauseHelper, that defines all errors, events, and document all functions? Just to make this file cleaner.

Reply:
In the same vein, is this confusable with Vault pausing? (i.e., will people think you can also pause the Vault with this?) Maybe it should be called `PausePoolHelper`.